packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -220,8 +220,6 @@ export default function VisualEditor( { styles } ) {
 							<AnimatePresence>
 								<motion.div
```

Comment:
Could you also remove this div and `AnimatePresence`?

Reply:
Aren't they needed for the canvas to be able to animate when you shift viewports or go into template editing mode? This is the full context:

```
							<AnimatePresence>
								<motion.div
									key={ isTemplateMode ? 'template' : 'post' }
								>
```

Reply:
You removed the animation for that `motion.div`, so it won't do anything anymore.